package.json

```diff
@@ -51,10 +51,15 @@
   },
   "packageManager": "yarn@3.2.3",
   "devDependencies": {
+    "@nomiclabs/hardhat-ethers": "^2.2.3",
+    "@nomiclabs/hardhat-waffle": "^2.0.6",
```

Comment:
These dependencies are not being used by your personal page, so I imagine you can remove them from the PR.